src/components/10-atoms/radio/CHANGELOG.md

```diff
@@ -1,13 +1,17 @@
 # Changelog
 
+### 2.0.2
+
+prevent doubled labels when `label` attribute isn't set
```

Comment:
`Prevent doubled labels when `label` attribute is not set.`

Reply:
✅